tools/api-markdown-documenter/src/renderers/html-renderer/configuration/RenderOptions.ts

```diff
@@ -0,0 +1,105 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import {
+	type BlockQuoteNode,
+	type CodeSpanNode,
+	type DocumentationNode,
+	DocumentationNodeType,
+	type FencedCodeBlockNode,
+	type HeadingNode,
+	type LinkNode,
+	type OrderedListNode,
+	type ParagraphNode,
+	type PlainTextNode,
+	type SectionNode,
+	type SpanNode,
+	type TableCellNode,
+	type TableNode,
+	type TableRowNode,
+	type UnorderedListNode,
+} from "../../../documentation-domain";
+import type { DocumentWriter } from "../../DocumentWriter";
+import type { RenderContext } from "../RenderContext";
+import {
+	renderBlockQuote,
+	renderCodeSpan,
+	renderFencedCodeBlock,
+	renderHeading,
+	renderHierarchicalSection,
+	renderHorizontalRule,
+	renderLineBreak,
+	renderLink,
+	renderOrderedList,
+	renderParagraph,
+	renderPlainText,
+	renderSpan,
+	renderTable,
+	renderTableCell,
+	renderTableRow,
+	renderUnorderedList,
+} from "../default-renderers";
+
+/**
+ * Configuration for rendering {@link DocumentationNode}s to `HTML`, specified by {@link DocumentationNode."type"}.
+ *
+ * @remarks
+ *
+ * Note: in order to be complete, this *must* include an entry for all types enumerated under {@link DocumentationNodeType}.
```

Comment:
Can we enforce that with typing of the keys for `Renderers`?

Reply:
There is probably a way, but I'm not sure it's worth it. The "DocumentationNode" domain is extensible, so we don't technically know what the complete set is. We know what the complete set of system-provided node kinds is, but we always inject our defaults unless overridden anyways. I might just update this comment to note that if we encounter a node kind without a renderer, we will throw an error.